pkg/webhooks/annotations.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 const (
-	policyAnnotation = "policies.kyverno.io~1patches"
+	policyAnnotation = "policies.kyverno.io~1last-applied-patches"
```

Comment:
@weastel - thanks for the work.

One question: what about those old resources that have annotation `policies.kyverno.io/patches`? Do we need to remove it?

Reply:
This annotations are generated after successful patches are applied. So these will not matter much, but deleting them will help to avoid confusion.

Reply:
Agreed! Will you make that change in this PR?

Reply:
Delete the old annotation with key `policies.kyverno.io/patches`, if present.

Reply:
@weastel - are you going to make this change?